PrebidMobile/Swift/AdUnits/Utils.swift

```diff
@@ -417,6 +418,18 @@ public class Utils: NSObject {
 
           return nil
       }
+    
+    @objc public func round(coordinates : CLLocationCoordinate2D, precision: UInt) -> CLLocationCoordinate2D {
```

Comment:
Please add more Unit tests for this method. Use as many variants as you can imagine, including corner cases and nonsense values, like precision = 1000000000 or precision = -1 (If you'll have to use another data for ObjC compatibility)